development/blog/pokemononabike.pug

```diff
@@ -9,20 +9,20 @@ head
 	meta(property="og:description"       	content="For you risky adventurous folk… here’s how I paired an awesome day trip bike ride with a Pokemon adventure")
```

Comment:
You should be separating this type of cleanup and other tasks into different branches. Stay focused on a task per branch to keep the code focused and clear. Your reviewers will thank you for it ;)